test/modules/TestResultCoordinator/Program.cs

```diff
@@ -25,7 +25,7 @@ static async Task Main(string[] args)
             await SetupEventReceiveHandlerAsync(Settings.Current, DateTime.UtcNow);
 
             // Continue after delay to report test results
-            Task.Delay(Settings.Current.TestDuration + Settings.Current.DurationBeforeVerification, cts.Token)
+            await Task.Delay(Settings.Current.TestDuration + Settings.Current.DurationBeforeVerification, cts.Token)
```

Comment:
revert this; we want to fire and continue to run next line to init storage and running web host.